js/apiv2.js

```diff
@@ -117,13 +117,21 @@ const api = {
             .then(response => response.statusText)
             .catch(this.logFailure)
     },
-    async downloadTemplate () {
-        // TODO
-            // return await this.templatesInstance.get('/')
-        console.log("API request to get Templates");
-        return await Promise.resolve({data: "EXCEL TEMPLATE"})
-            .then(response => response.data)
-            .catch(this.logFailure);
+    async downloadTemplate (program_id) {
+            return await this.templatesInstance.get(`/indicators/bulk_import_indicators/${program_id}/`)
+            .then(response => {
+                const url = window.URL.createObjectURL(new Blob([response.data]));
```

Comment:
Nice, treating as a blob is what I was missing when I tried the link create method.

Reply:
Cool, we were thinking along the same lines then.